src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -116,6 +116,7 @@
   "mhvSecureMessagingToVaGovRelease": "mhv_secure_messaging_to_va_gov_release",
   "mhvSecureMessagingToPhase1": "mhv_secure_messaging_to_phase_1",
   "mhvSecureMessagingCernerPilot": "mhv_secure_messaging_cerner_pilot",
+  "mhvSecureMessagingUniqueUserLoggingEnabled":"mhv_secure_messaging_unique_user_logging_enabled",
```

Comment:
guessing  [features.yml](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/features.yml) is still pending addition of this flag?

Reply:
Correct. The feature flag PR is still to be created.

Reply:
PR is created. https://github.com/department-of-veterans-affairs/vets-api/pull/17015